ccvm_simulators/solvers/dl_solver.py

```diff
@@ -482,7 +429,10 @@ def _solve_adam(
                 Defaults to None, which generates a filename based on the problem instance name.
```

Comment:
`post_processor` argument can be removed from the function as well as the docstirngs. Same for all the solvers.